todo-list-service/src/main/java/org/example/todo_list_service/TodoListServiceApplication.java

```diff
@@ -0,0 +1,36 @@
+package org.example.todo_list_service;
+
+import java.util.ArrayList;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+public class TodoListServiceApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(TodoListServiceApplication.class, args);
+	}
+
+	@RestController
+	public class HelloController {
+
+		@GetMapping("/hello")
+		public String sayHello(String name, String lastName) {
+			return "¡Hello World!";
+		}
+
+		@GetMapping("/todos")
```

Comment:
Extract this into endpoint a new class and place it in a new file too. We can call the new class `TodosController` or something like that.